common/src/cursor.rs

```diff
@@ -103,9 +102,10 @@ impl std::ops::Neg for Dir {
     }
 }
 
-lazy_static! {
-    /// Maps every (A, B, C) sharing a vertex to A', the side that shares edges with B and C but not A
-    static ref NEIGHBORS: [[[Option<Side>; SIDE_COUNT]; SIDE_COUNT]; SIDE_COUNT] = {
+/// Maps every (A, B, C) sharing a vertex to A', the side that shares edges with B and C but not A
+fn neighbors_static() -> &'static [[[Option<Side>; SIDE_COUNT]; SIDE_COUNT]; SIDE_COUNT] {
```

Comment:
`_static` is an unhelpful suffix. Anything returned by a free function must necessarily have a static lifetime. Did you actually encounter any collisions when leaving it off?

Reply:
I did:

```
error[E0599]: no method named `iter` found for unit type `()` in the current scope
   --> common/src/dodeca.rs:460:38
    |
460 |         let triples = vertex_sides().iter().collect::<HashSet<_>>();
    |                                      ^^^^ method not found in `()`

error[E0277]: `()` is not an iterator
   --> common/src/dodeca.rs:462:24
    |
462 |         for &triple in vertex_sides() {
    |                        ^^^^^^^^^^^^^^ `()` is not an iterator
    |
    = help: the trait `Iterator` is not implemented for `()`, which is required by `(): IntoIterator`
    = note: required for `()` to implement `IntoIterator`
```

Reply:
Rename the test function to `vertex_sides_consistent()` or something.